time stamp
int64 num_rois
int32[] roi_detected_points
int32[] roi_threshold_used
float32[] roi_kl_divergence
int32[] roi_is_valid
